metadata/demographic/census_places_2020.md

```diff
@@ -0,0 +1,77 @@
+
+## Description
+
+### What is the dataset?
+
+This dataset contains polygon boundaries for cities, towns, metro townships, and census-designated places (CDPs) in Utah. The [US Census Bureau](https://www.census.gov/programs-surveys/bas/information/cdp.html) defines CDPs as "statistical geography representing closely settled, unincorporated communities that are locally recognized and identified by name." Both these unincorporated communities and legally incorporated areas of development are represented in this dataset.
+
+### What is the purpose of the dataset?
+
+The US Census Bureau uses both legally incorporated (cities, towns, etc) and unincorporated places (CDPs) for the once-a-decade count of population and housing. While this dataset does not contain demographic information, it can be joined to other layers to visualize population data.
+
+Geography has always been a fundamental aspect of the census. You can read more about how geography is used in the census on the Kem C. Gardner Policy Institute's 2020 Census Geography [Blog](https://gardner.utah.edu/blog/blog-whats-new-in-utahs-census-2020-geography/) and [fact sheet](https://d36oiwf74r1rap.cloudfront.net/wp-content/uploads/Geog-FS-Mar2021.pdf).
+
+<!--- Are the data also involved in the redistricting process? -->
+
+### What does the dataset represent?
+
+Incorporated cities and CDPs are represented as polygons and can be distinguished by the CLASSFP20 field. Each polygon represents the geographic extent of a given place and includes the place name and type.
+
+<!--- I'm planning on including more information about the CLASSFP20 field once I hear back from Greg, but are there other fields that would be helpful to include here? -->
+
+### How was the dataset created?
+
+Utah's 2020 census geographic boundaries became available in February 2021 and the demographic redistricting legacy files (the 2020 Census Redistricting P.L. 94-171 Summary Files) became available on August 12, 2021. The Census Bureau released a user-friendly version of the 2020 Census Redistricting P.L. 94-171 Summary Files on September 30, 2021. UGRC, in collaboration with the Analytics Group at [WFRC](https://wfrc.org/) used that August 12 release to create ready-to-use GIS data in several formats. You can learn more about these offerings on [our website](https://gis.utah.gov/blog/2021-08-31-census-2020-redistricting-data/).
+
+### How reliable and accurate is the dataset?
+
+This dataset is the most current and complete dataset for incorporated municipalities and CDPs in Utah. All 2020 census geographies reflect boundaries as of January 1, 2020. Additional GIS files and support products are available on the US Census Bureau [Redistricting Data Page](https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2).
```

Comment:
Digging in, we have muni and metro township boundaries (polygons) and city/town points, which makes this our only unincorporated place boundary dataset. But I agree, I think the muni boundaries are the ones we should be advertising as most current and complete. Does this dataset get updated throughout the decade, or only at each decennial census?

Reply:
at the moment, we only update this data on the decennial cycle. this dataset targets a specific user who is specifically looking for census-based data. otherwise, yeah, we would steer them to the political boundary layers that we create and provide.  
agreed, this is the only layer with census-designated place. i'm not sure if it gets updated more frequently than every 10 years. if so, it would be int he ACS datasets.